pontoon/teams/views.py

```diff
@@ -122,13 +122,11 @@ def ajax_permissions(request, locale):
     translators = l.translators_group.user_set.exclude(pk__in=managers).all()
     all_users = User.objects.exclude(pk__in=managers).exclude(pk__in=translators).exclude(email='')
 
-    contributors = (
-        User.translators
-        .filter(translation__locale=l)
-        .values_list('email', flat=True)
-        .distinct()
-    )
+    contributors_qs = User.translators.with_translation_counts(None, Q(locale=l))
```

Comment:
[nit/question] From what I understood `with_translation_count` will return only 100 contributors sorted by translation count?
Is that enough for every locale?

Reply:
Good call! I'll fix that!